app/views/spree/admin/users/_form.html.haml

```diff
@@ -0,0 +1,26 @@
+.row
+  .alpha.five.columns
+    = f.field_container :email do
+      = f.label :email, Spree.t(:email)
+      = f.email_field :email, class: "fullwidth"
+      = error_message_on :user, :email
+    .field
+      = label_tag nil, Spree.t(:roles)
+      %ul
+        - @roles.each do |role|
+          %li
+            = check_box_tag "user[spree_role_ids][]", role.id, @user.spree_roles.include?(role), id: "user_spree_role_#{role.name}"
+            = label_tag role.name
+      = hidden_field_tag "user[spree_role_ids][]", ""
+    = f.field_container :enterprise_limit do
+      = f.label :enterprise_limit, t(:enterprise_limit)
+      = f.text_field :enterprise_limit, class: "fullwidth"
+  .omega.five.columns
+    = f.field_container :password do
+      = f.label :password, Spree.t(:password)
```

Comment:
Is the admin user setting the user's password for them here? How are they going to tell the user what their password is? I think I ran into an issue like this before with emails and there was a way to set a random secure password, and flag the account as requiring a password reset. When the user followed the sign up link via their email they were taken straight to a "set your password" page by default via devise...

Reply:
Excellent question. Dannielle actually covered that:
> ##### Step 1(a): Super Admin signs up the user
> The Super Admin accesses the add a user function in the backend and submits that user's email address and sets a password (existing functionality).
>
> Note: we know that setting a password for a user is bad, and we've created #1612 to change this functionality so that the user sets it themselves.

So it's existing functionality that we are planning to tackle another time.